editor/src/core/property-controls/property-controls-utils.ts

```diff
@@ -397,14 +400,19 @@ export function getPropertyControlsForTarget(
 
       let filenameForLookup: string | null = null
       if (importedFrom == null) {
-        if (isIntrinsicHTMLElement(element.name)) {
-          /**
-           * We detected an intrinsic HTML Element (such as div, a, span, etc...)
-           * for the sake of simplicity, we assume here that they all support the style prop. if we need more detailed
-           * information for them, feel free to turn this into a real data structure that contains specific props for specific elements,
-           * but for now, I just return a one-size-fits-all PropertyControls result here
-           */
-          return HtmlElementStyleObjectProps
+        if (isIntrinsicElement(element.name)) {
+          if (isIntrinsicHTMLElement(element.name)) {
+            /**
+             * We detected an intrinsic HTML Element (such as div, a, span, etc...)
+             * for the sake of simplicity, we assume here that they all support the style prop. if we need more detailed
+             * information for them, feel free to turn this into a real data structure that contains specific props for specific elements,
+             * but for now, I just return a one-size-fits-all PropertyControls result here
+             */
+            return HtmlElementStyleObjectProps
+          } else {
+            // you can add more intrinsic (ie not imported) element types here
```

Comment:
I believe R3F elements (e.g. `mesh`) will fall into this category as they're not explicitly imported (e.g. in https://codesandbox.io/s/rrppl0y8l4?file=/src/App.js)